views/index.erb

```diff
@@ -0,0 +1,4 @@
+<h1>Makers BnB</h1>
+<form method="post" action="/sign-in">
```

Comment:
It can — but it's vital to remember to change it to a `post` when submitting this info. Personally I'd err on the side of caution and keep it as a post.